build.gradle

```diff
@@ -23,3 +24,35 @@ allprojects {
 
     apply plugin: 'com.github.ben-manes.versions'
 }
+
+def gitTag() {
+    def tag = 'git tag -l --points-at HEAD'.execute((List) null, rootProject.projectDir).text.trim()
```

Comment:
Otherwise there is a method call ambiguity because of overload that accepts array of strings. 

First param is list of env variables for the process, if it's `null` process inherits them from parent one.